app/views/users/new.html.erb

```diff
@@ -94,7 +113,7 @@
       |,
       data: { turbo_submits_with: "Creating..." }
     %>
-  <% end # password_authentication? %>
+<% end # password_authentication? %>
```

Comment:
same indentation question here. I don’t feel that strongly, but mostly I’d keep it just for consistency across the code base. I could easily see another contributor coming along and re-indenting all these in a future PR. :)